src/Migrator.sol

```diff
@@ -52,23 +54,21 @@ contract Migrator is IMigrator, Initializable, Ownable {
     /// @inheritdoc IMigrator
     function initialize(InitializationParams memory params) external initializer onlyOwner {
         weth = IWETH9(params.weth);
-        token = IERC20(params.token);
-        bpt = IERC20(params.bpt);
-        auraBpt = IERC20(params.auraBpt);
+        token = ERC20(params.token);
+        bpt = ERC20(params.bpt);
+        auraBpt = ERC20(params.auraBpt);
```

Comment:
nit: I'd recommend writing out variable names fully because without looking at documentation, this is difficult to understand what `auraBpt` stands for.

Reply:
updated